### DISCUSSION TITLE
Integrating PHPSimpleSAML [ADFS/Google] with Shield

### DISCUSSION THREAD

**Discussion by mauliknwdco** [codeigniter4/shield#426] [Ideas]
Is there a way that Shield can be integrated with PHP Simple SAML for authentication with ADFS / Google?

**Comment by WaaaghNL** [comment]
Not that i know but i love the idea!

**Comment by datamweb** [comment]
Hello @mauliknwdco , do you mean the login button with Google in the shield?
